google/cloud/forseti/common/gcp_type/billing_account.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A Billing Account Resource."""
+
+from google.cloud.forseti.common.gcp_type import resource
+
+
+class BillingAccountLifecycleState(resource.LifecycleState):
+    """Represents the Billing Account's LifecycleState."""
+    pass
+
+
+class BillingAccount(resource.Resource):
+    """BillingAccount Resource."""
+
+    RESOURCE_NAME_FMT = 'billingAccounts/%s'
+
+    def __init__(
+            self,
+            billing_account_id,
+            full_name=None,
+            data=None,
+            name=None,
+            display_name=None,
+            parent=None,
+            lifecycle_state=BillingAccountLifecycleState.UNSPECIFIED):
```

Comment:
Not a Python expert but it seems this is a mutable default arg and might be an anti-pattern?

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html